src/Functions/FunctionsCoding.h

```diff
@@ -1506,6 +1507,95 @@ class FunctionBitmaskToArray : public IFunction
     }
 };
 
+class FunctionBitPositionsToArray : public IFunction
+{
+public:
+    static constexpr auto name = "bitPositionsToArray";
+    static FunctionPtr create(ContextPtr) { return std::make_shared<FunctionBitPositionsToArray>(); }
+
+    String getName() const override
+    {
+        return name;
+    }
+
+    size_t getNumberOfArguments() const override { return 1; }
+    bool isInjective(const ColumnsWithTypeAndName &) const override { return true; }
+
+    DataTypePtr getReturnTypeImpl(const DataTypes & arguments) const override
+    {
+        if (!isNativeInteger(arguments[0]))
```

Comment:
Not native may be also ok?

Reply:
We can also extend bitmask functions for 128, 256 bits.

Reply:
Fixed.